example/src/screens/Examples/AwareScrollViewStickyFooter/StickyFooter.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { memo, useMemo, type ReactNode } from 'react';
+import { ViewProps, type ViewStyle } from 'react-native';
+import { useReanimatedKeyboardAnimation } from 'react-native-keyboard-controller';
+import Animated, {
+  interpolate,
+  useAnimatedStyle,
+} from 'react-native-reanimated';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+export interface StickyFooterProps extends Pick<ViewProps, 'onLayout'> {
+  children: ReactNode;
+  /**
+   * The offset to add when the keyboard is closed. Useful to position the
+   * sticky bar above things such as a tab bar.
+   */
+  bottomOffsetWhenKeyboardIsClosed?: number;
+}
+
+export const StickyFooter = memo(
+  ({
+    children,
+    bottomOffsetWhenKeyboardIsClosed,
+    ...props
+  }: StickyFooterProps) => {
+    const { height, progress } = useReanimatedKeyboardAnimation();
+    const { bottom } = useSafeAreaInsets();
+
+    const actionBarAnimatedStyles = useAnimatedStyle(() => {
+      const translateY = interpolate(progress.value, [0, 1], [0, height.value]);
+
+      /**
+       * When the keyboard opens, we don't want additional padding since things
+       * like tab bar and safe area are not visible.
+       */
+      const paddingBottom = interpolate(
+        progress.value,
+        [0, 1],
+        [bottomOffsetWhenKeyboardIsClosed ?? 0, 0]
```

Comment:
The default value can be assigned to `0` and then here we can use `[bottomOffsetWhenKeyboardIsClosed, 0]`, right? In this case it'll work slightly faster, because we will not have a conditional statement of each frame change :)